moveit_core/utils/CMakeLists.txt

```
@@ -7,7 +7,12 @@ target_include_directories(moveit_utils PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/moveit_core>
 )
+<<<<<<< HEAD
 ament_target_dependencies(moveit_utils Boost moveit_msgs)
+=======
+ament_target_dependencies(moveit_utils Boost moveit_msgs rclcpp fmt)
+target_link_libraries(moveit_utils rsl::rsl)
+>>>>>>> 3df6b2813 (Depend on `rsl::rsl` as a non-Ament dependency (#2578))
```

```suggestion
ament_target_dependencies(moveit_utils Boost moveit_msgs)
```
In this branch `moveit_utils` doesn't depend on `rsl::rsl` so the original can be kept.